rust/src/tor.rs

```diff
@@ -191,19 +198,61 @@ impl LwdConn {
     /// Calls the given closure with the transactions corresponding to the given t-address
     /// within the given block range, and the height of the main-chain block they are
     /// mined in (if any).
-    pub(crate) fn with_taddress_transactions(
+    pub(crate) fn with_taddress_utxos(
         &mut self,
         params: &impl consensus::Parameters,
         address: TransparentAddress,
         start: Option<BlockHeight>,
+        limit: u32,
```

Comment:
This would leak fewer gRPC internal details:
```suggestion
        limit: Option<u32>,
```